library/modules/Items.cpp

```diff
@@ -1756,7 +1759,34 @@ bool Items::createItem(vector<df::item *> &out_items, df::unit *unit, df::item_t
     for (auto out_item : out_items)
     {   // Plant growths need a valid "growth print", otherwise they behave oddly
         if (auto growth = virtual_cast<df::item_plant_growthst>(out_item))
+        {
+            int growth_print = -1;
+            // Make sure it's made of a valid plant material, then grab its definition
+            if (growth->mat_type >= 419 && growth->mat_type <= 618 && growth->mat_index >= 0 && (unsigned)growth->mat_index < world->raws.plants.all.size())
```

Comment:
I am horrified that we don't have constants defined for these, but looking through the code, I see these numbers copied into many places.

Reply:
I think we _do_ have them in the Materials module (as `MaterialInfo::PLANT_BASE`), but I couldn't be bothered to look it up at the time. Besides, I'm eventually going to add all of the individual enum values to structures, so I'll clean them up here when that happens..